build.gradle

```diff
@@ -907,12 +908,29 @@ dependencyUpdates.resolutionStrategy {
 }
 
 // Support for creating IntelliJ IDEA files.
-// TODO This isn't working presently
 idea {
+    project {
+        def javafxLib = file("$projectDir/mods/lib")
+        def files = javafxLib.listFiles( { _, name -> name.contains(".jar") } as FilenameFilter)
+                .toList() // no direct way to convert from Array to Set
+                .toSet()
+
+        def projectLibrary = new ProjectLibrary()
+        projectLibrary.name = "lib"
+        projectLibrary.classes += files
+
+        projectLibraries += [ projectLibrary ]
+    }
+
     module {
-        testSourceDirs += project.sourceSets.itest.java.srcDirs
-        testSourceDirs += project.sourceSets.slowtest.java.srcDirs
-        testSourceDirs += project.sourceSets.test.java.srcDirs
+        testSources.from(project.sourceSets.itest.java.srcDirs,
```

Comment:
Is this missing `project.sourceSets.test.java.srcDirs`?

Reply:
Nice catch, it is because the test folder is filled with this block. It seems, the IDEA plugin is smart enough to use it:
```groovy
    // Running regular tests
    test {
        java {
            srcDirs = ['code/src/utest', 'code/src/testcommon']
        }
        compileClasspath += sourceSets.main.output
        runtimeClasspath += sourceSets.main.output
        resources {
            srcDirs = ['code/src/testResources']
        }
    }
```

Reply:
Here is the fresh project, generated by Gradle:
<img width="291" alt="Sources in the Module" src="https://user-images.githubusercontent.com/1671049/229441757-1c651b26-8e70-4335-8c06-fb10db42005d.png">
Look, the `utest` folder is there.